### PR TITLE
fix(suite-native): enable discovery for all device supported networks

### LINKED ISSUE #10456: discovery of deprecated networks
The PR https://github.com/trezor/trezor-suite/pull/10241 refactoring supporedNetworks configuration file made changes, that disable discovery of deprecated coins such Vertcoin, Bitcoin Gold, DigiByte etc.

Acceptance criteria:
- If you connect the T1B1 or T2T1 device, the discovery is started for all coins supported by these devices (including the deprecated ones)

### PR DESCRIPTION
**Pull request by vytick**
Discovery happened only for some networks. This PR should allow all device supported accounts minus blacklisted ones

Resolves #10456

### CODE REVIEW

File: suite-native/discovery/src/discoveryThunks.ts [hunk 1]
```diff
@@ -318,19 +314,19 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
...
+        const supportedNetworks = sortNetworks(getAvailableNetworks(enabledNetworks, device));
 
         // Start tracking duration for analytics purposes
         dispatch(setDiscoveryStartTimestamp(performance.now()));
 
         await dispatch(
             createDescriptorPreloadedDiscoveryThunk({
                 deviceState,
-                deviceSupportedNetworks,
+                deviceSupportedNetworks: supportedNetworks,
```
Comment (PeKne) [inline suite-native/discovery/src/discoveryThunks.ts]: nit: Could you please improve the naming little bit? It is confusing to me. The value is originally stored in `const deviceSupportedNetworks`, then it is filtered out and the value is stored to `const enabledNetworks`.  `enabledNetworks` is filtered again and result stored in `supportedNetworks`. And in the end it is again mapped to the `deviceSupportedNetworks` prop at this line. 

What about using `pipe()` form ts-belt, in a way (just example):

```ts
   const deviceSupportedNetworks = pipe(selectDeviceSupported, filterTestnets, filterOutUnavailableAndBlacklisted)
```

**Review by PeKne — APPROVED** [review]

File: suite-native/discovery/src/discoveryThunks.ts (hunk 1 above)
Comment (vytick) [reply suite-native/discovery/src/discoveryThunks.ts]: What about this way?

93d58d9770

Comment (PeKne) [reply suite-native/discovery/src/discoveryThunks.ts]: Yes, great 👍. Thanks for reworking this.

File: suite-native/state/src/extraDependencies.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ const transports = transportsPerDeviceType[deviceType];
 
 export const extraDependencies: ExtraDependencies = mergeDeepObject(extraDependenciesMock, {
     selectors: {
-        selectEnabledNetworks: () => supportedNetworkSymbols,
+        selectEnabledNetworks: () => supportedPortfolioTrackerNetworkSymbols,
```
Comment (Nodonisko) [inline suite-native/state/src/extraDependencies.ts:28]: @vytick Why did you put portfolio tracker coins here? This selector shoudn't be related to portfolio tracker.

**Comment by vytick** [comment]
it is still the same thing that was there before. I have just renamed it so
I had to reference it here with a new name.

**Pull request merged**